### PR TITLE
Removed error catching

### PR DESCRIPTION
**Pull request by JoelLucaAdams**

### CODE REVIEW

File: demobot.py [hunk 1]
```diff
@@ -45,22 +45,5 @@ async def on_ready():
...
-async def on_command_error(ctx, error):
-    """
-    Handle the Error message in a nice way.
-    """
-    if isinstance(error, commands.errors.CheckFailure):
-        await ctx.send('You do not have the correct role for this command.')
-    elif isinstance(error, commands.errors.MissingRequiredArgument):
-        await ctx.send('You are missing a required argument.')
-    elif isinstance(error, commands.errors.CommandNotFound):
-        await ctx.send('Sorry, I didn\'t get that.')
```
Comment (IdrisTheDragon) [inline demobot.py:59]: Only remove this line not all the error checking. Leave the elif case in though so it doesn't do the default case for any other error.

**Review by IdrisTheDragon — CHANGES_REQUESTED** [review]
See inline comments

File: demobot.py (hunk 1 above)
Comment (JoelLucaAdams) [reply demobot.py:59]: problem is now the elif doesn't do anything and the elif fails

Comment (IdrisTheDragon) [reply demobot.py:59]: Oh ye python can't have it empty, so try add a `pass` into it.

**Review by IdrisTheDragon — APPROVED** [review]

**Pull request merged**